main.js

```diff
@@ -2,14 +2,14 @@ var winsByComputer = 0;
 var winsByPlayer = 0;
 var computerWeapon = (Math.random());
```

Comment:
Check please, how many times do you generating the random numbers during the game.